app/models/feedbook/answer.rb

```diff
@@ -0,0 +1,6 @@
+module Feedbook
+  class Answer < ApplicationRecord
+    belongs_to :question
+    belongs_to :employees_session
+  end
+end
```

Comment:
it's advisable to leave a blank line at the end of the file

Reply:
Many style check tools like `rubocop`, `reek` HounCI, etc, will complain about not having a blank line at the end of the file. Most editors have a feature you enabled so they add the blank line automatically at saving.

Reply:
Thank you, it's funny because I do have a blank line at line 7, but I can't tell why doesn't show it here...

Reply:
It's ok @diegosaul402, github does not show blank lines at the end of the files, if there's a missing line it shows an alert next to the last line